### DISCUSSION TITLE
League issue [SOLVED]

### DISCUSSION THREAD

**Discussion by Bloodspor7** [masterplane8/F1-League-Management#21] [Q&A] [answered]
Dear,
i've got an issue where 4 drivers finished +2 laps but the document is putting them all in 6th place. How do i fix this ?
![Screenshot_1](https://user-images.githubusercontent.com/112808570/188312276-207f14b4-bb0e-4a07-b951-3f0b82bd6a7e.png)

**Comment by Crisjeeeee** [comment]
xD

**Comment by olewis14** [comment, marked as answer]
This discussion says it is solved, but I just want to point out the solution anyway, At the top of the Final Classifications page is directions on how to enter information into the Time From Leader columns, including how to indicate the order of drivers who finish laps down from the leader.
